x/examples/outline-connectivity/main.go

```diff
@@ -206,34 +182,31 @@ func main() {
 				DurationMs: testDuration.Milliseconds(),
 				Error:      makeErrorRecord(testErr),
 			}
-			err := jsonEncoder.Encode(record)
+			u, err := url.Parse(*reportToFlag)
```

Comment:
I'd prefer a clearer name for readability:
```suggestion
			collectorURL, err := url.Parse(*reportToFlag)
```